molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java

```diff
@@ -1,4 +1,4 @@
-package org.molgenis.data.annotation.cmd;
+package org.molgenis.annotation.cmd;
```

Comment:
/**
  * Build JAR file...............: mvn clean install -pl molgenis-data-annotators/ -am -DskipTests -P create-delivery
  * Run..........................: java -jar molgenis-data-annotators/target/CmdLineAnnotator.jar
  */

Change javadoc example commands to correct module paths